src/PublicApiGeneratorTests/PublicApiGeneratorTests.csproj

```diff
@@ -25,8 +25,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="xunit" Version="2.5.2" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
+    <PackageReference Include="xunit" Version="2.6.2" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.4" NoWarn="NU1701">
```

Comment:
```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error	NU1701	Warning As Error: Package 'xunit.runner.visualstudio 2.5.4' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework '.NETCoreApp,Version=v3.1'. This package may not be fully compatible with your project.	PublicApiGeneratorTests	C:\_GITHUB_\PublicApiGenerator\src\PublicApiGeneratorTests\PublicApiGeneratorTests.csproj	1		
```